### ISSUE TITLE
Jobs remaining in the 'executing' state in the agent

### ISSUE THREAD

**Issue by amaltaro** [dmwm/WMCore#6969]
It's not so common, but we have been seen lately jobs that are gone but still marked as 'executing' in wmbs, e.g. in submit2:
https://cms-logbook.cern.ch/elog/Workflow+processing/24865

I can find the condor logs under JobCreator (with logging from the worker node) but there is no trace of these jobs in JobAccountant (as expected). To be investigated...

**Comment by amaltaro** [comment]
Just updating here while the debugging is on going. I found one case in submit1 where 1 `executing` job belongs to an `aborted-archived` workflow. The jobid in wmbs is `3643803` for `prozober_ACDC_task_B2G-RunIISpring16DR80-00351__v1_T_160510_174036_8226`.

ahaa, here is what I found in JobStatusLite
```
2016-05-23 05:47:26,225:140131428202240:INFO:StatusPoller:Killing job 3643803 because it has exceeded timeout for status Error
2016-05-23 05:47:31,344:140131428202240:ERROR:BossAirAPI:Could not write a kill FWJR due to non-existant cache_dir for job 4242017

2016-05-23 05:47:31,355:140131428202240:ERROR:PyCondorPlugin:Serious Error in Completing condor job with id 4242017!
Could not find jobCache directory - directory deleted under job: /storage/local/data1/cmsdataops/srv/wmagent/v1.0.14.patch2/install/wmagent/JobCreator/JobCache/prozober_ACDC_task_B2G-RunIISpring16DR80-00351__v1_T_160510_174036_8226/B2G-RunIISpring16DR80-00351_1/B2G-RunIISpring16DR80-00351_1MergeRAWAODSIMoutput/JobCollection_410809_0/job_3643803
Creating artificial cache_dir for failed job report
```
now onto debugging active workflows... and then I'll fix those issues.

**Comment by amaltaro** [comment]
And then a slightly different error for an active workflow, e.g.  from submit2 `vlimant_task_SUS-RunIIFall15DR76-00120__v1_T_160609_143246_6395` for job wmbsid `1374949`. Logs for JobStatusLite
```
2016-06-17 21:31:14,590:140065338152704:INFO:StatusPoller:Killing job 1374949 because it has exceeded timeout for status 'Pending'
2016-06-17 21:31:14,696:140065338152704:ERROR:BossAirAPI:Could not retrieve job with WMBS ID 1374949 from BossAir database
2016-06-17 21:31:14,696:140065338152704:ERROR:BossAirAPI:Could not load all jobs in BossAir for WMBS input
```

so there is a flaw in the condor timeout kill.

**Comment by amaltaro** [comment]
Fixed by #6977
